projects/storefrontapp-e2e-cypress/cypress/helpers/product-configuration-overview.ts

```diff
@@ -111,3 +111,64 @@ export function verifyNotificationBannerOnOP(numberOfIssues?: number): void {
     element.get('.cx-error-msg').should('not.be.visible');
   }
 }
+
+/**
+ * Verifies whether the group header displayed.
+ */
+export function checkGroupHeaderDisplayed(
+  groupName: string,
+  groupIdx: number
+): void {
+  cy.get('cx-configurator-overview-form .cx-group h2')
+    .eq(groupIdx)
+    .should('contain.text', groupName);
+}
+
+/**
+ * Verifies whether the group header displayed.
```

Comment:
correct description: ...is not displayed.

Reply:
Method is never used, is it required?

Reply:
there was another bug(displaying empty groups), which is now merged and now also the method is called